Code/STAGE1bih_merging_and_organization.R

```diff
@@ -0,0 +1,49 @@
+## Jericho Lawson
+## Summer 2019, 2021
+## Merging and Organization
+
+# Adds calculated features to the data gathered from the MIT-BIH/2017 subjects.
+# Merges all data into one file.
+
+## Pick the correct version for testing. ##
+VERSION = "M"
+#VERSION = "C"
+#VERSION = "multi_2017"
+
```

Comment:
file_path = "C:/Users/mario/Documents/Internship and REU Stuff/UNCW 2019/Atrial_Fibrillation/Testing"